tests/load/locustfile.py

```diff
@@ -0,0 +1,405 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import base64
+import json
+import time
+import uuid
+from contextlib import closing
+from urllib.parse import urljoin, urlparse
+
+from locust import HttpUser, TaskSet, events, task
+from websocket import create_connection
+from websocket._exceptions import WebSocketTimeoutException
+
+
+@events.init_command_line_parser.add_listener
+def _(parser):
+    parser.add_argument(
+        "--websocket_url",
+        type=str,
+        env_var="AUTOPUSH_WEBSOCKET_URL",
+        default="",
```

Comment:
```suggestion
        required=True,
```
From what i understand, we need these, and they really shouldn't be blank. Suggest that we make them required instead.